Mathlib/NumberTheory/ModularForms/JacobiTheta/Basic.lean

```diff
@@ -100,9 +100,11 @@ theorem jacobiTheta_S_smul (τ : ℍ) :
   rw [mul_comm ((1 : ℂ) / _) _, mul_one_div, eq_div_iff ha', mul_comm _ (_ ^ _), eq_comm] at this
   have expo1 : ∀ n : ℤ, -↑π / (-I * ↑τ) * (n : ℂ) ^ 2 = ↑π * I * (n : ℂ) ^ 2 * (-↑τ)⁻¹ := by
     intro n
-    field_simp [hτ, I_ne_zero]
+    simp only [neg_mul, neg_div_neg_eq, div_mul_eq_mul_div, inv_eq_one_div, mul_div_assoc', mul_one,
```

Comment:
Should this be viewed as a regression? Or is there an explanation why this `simp only` is morally the better proof?

Reply:
Before it was nonterminal so liable to break

Reply:
But I'm not sure we have a field simp only